source/classes.tex

```diff
@@ -1873,7 +1866,9 @@
 \end{codeblock}
 
 \pnum
-A user-declared move assignment operator \tcode{X::operator=} is
+\indextext{move!class object|see{assignment operator, move}}%
+\indextext{operator!move assignment|see{assignment operator, move}}%
+A \defnadj{move}{assignment operator} \tcode{X::operator=} is
 a non-static non-template member function of class \tcode{X} with exactly
```

Comment:
I don't like that for copy constructors, `X` is first introduced as

> A non-template constructor for class `X`

... but down here, it's phrased as

> ... is a non-static non-template member function of class `X`

Surely, we can make the wording for all these special member functions more symmetrical, and be more consistent about whether `X` or `X::operator=` is introduced first.

It also feels a bit redundant that we need to state that `operator=` is a non-static member function. Shouldn't that be obvious from its code spelling?

Reply:
We're also inconsistent about "for class" vs. "of class".

Reply:
We're inconsistent about a lot of things across the standard.  This seems like a pure indexing improvement and thus can be reviewed/applied a lot quicker than changes beyond that.  Don't do that here.

Reply:
To be fair, it's not just a pure indexing change. The PR also removes the term "user-declared" from wording. I wouldn't have said anything if that wasn't the case. I can live with the consistency tweaks being done separately though.

Reply:
Removal of "user-declared" was planned for discussion in 2017 (https://github.com/cplusplus/draft/issues/556#issuecomment-283569920). I think this can be applied together (otherwise we might need another issue for it).